Firestore/Source/Local/FSTLocalDocumentsView.mm

```diff
@@ -159,6 +171,26 @@ - (DocumentMap)documentsMatchingDocumentQuery:(const ResourcePath &)docPath {
   return result;
 }
 
+- (DocumentMap)documentsMatchingCollectionGroupQuery:(FSTQuery *)query {
+  HARD_ASSERT(query.path.empty(),
+              "Currently we only support collection group queries at the root.");
+  std::string collection_id = MakeString(query.collectionGroup);
+  std::vector<ResourcePath> parents = _indexManager->GetCollectionParents(collection_id);
+  DocumentMap results;
+  // Perform a collection query against each parent that contains the collection_id and
```

Comment:
Optional: consider adding a newline before this line, to visually separate the long `for` loop.